docker/ui/settings.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <profiles>
+        <profile>
+            <id>syndesisRepo</id>
+            <repositories>
+                <repository>
+                    <id>fuse-all</id>
+                    <name>fuse-all</name>
+                    <url>http://nexus.fuse-qe.eng.rdu2.redhat.com/repository/fuse-all/</url>
```

Comment:
is it needed? it would be good not to leak the internal stuff

Reply:
it is needed for apicurio UI steps dependency. However,
🤔 the whole commit is old (I don't use settings.xml anymore), probably I didn't push the actual local rebase (I will fix it tomorrow). Anyway, when you look to the backport ( https://github.com/syndesisio/syndesis-qe/pull/1566/commits/2a19c68bc45bfeda02e062a72bab1e6ab5809e80#diff-f1a049876079bf56678d719d5a76e41fR76 ), I use that repo only for the apicurio artifact so the other dependencies are downloaded from the public central. Is that way ok?

Reply:
you can build apicurio the same way you are building syndesis and copy it in the local repo

Reply:
I didn't know that apicurio tests are in github, ok, I will try to use it.